packages/app/src/vis-packs/core/heatmap/utils.ts

```diff
@@ -0,0 +1,10 @@
+import type { TextureTypedArray } from '@h5web/lib/src/vis/heatmap/models';
```

Comment:
Should we expose the type `TextureTypedArray` in `@h5web/lib` to clean up the import ?